server/src/main/java/online/dipa/hub/services/TimelineService.java

```diff
@@ -181,21 +181,21 @@ private void initializeMilestones(final Long timelineId) {
         for (Increment increment : incrementsList) {
             List<Milestone> incrementMilestones = new ArrayList<Milestone>();
 
-            long newDaysBetween = DAYS.between(increment.getStart(), increment.getEnd());
+            long newDaysBetween = DAYS.between(increment.getStart().plusDays(14), increment.getEnd());
```

Comment:
@navismo 

ich würde für eine bessere wartbarkeit inc.getStart.plusDay in eine lokale constante speichern.

Reply:
@deleonio Danke für den Vorschlag, ich habe das in #60 aufgenommen